test/ocamlorg_web/graphql_test.ml

```diff
@@ -145,47 +230,113 @@ let state_test () =
 let () =
   Alcotest.run
     "ocamlorg"
-    [ ( "test that starts_with function compares two similar strings"
+    [ ( "test that get_info function returns all the dependencies"
+      , [ Alcotest.test_case
+            "and returns all 6 dependencies"
+            `Quick
+            get_info_test
+        ] )
+    ; ( "test that is_in_range function checks if a current version is within \
+         the start and last version range specified"
       , [ Alcotest.test_case
             "and returns true"
             `Quick
-            (start_with_test "OC" "oc" true)
+            (is_in_range_test
+               (Package.Version.of_string "3.12.0")
+               (Some (Package.Version.of_string "2.0.0"))
+               (Some (Package.Version.of_string "5.9.2"))
+               true)
         ] )
-    ; ( "test that starts_with function compares two different strings"
+    ; ( "test that is_in_range function checks if a current version is outside \
+         the start and last version range specified"
       , [ Alcotest.test_case
             "and returns false"
             `Quick
-            (start_with_test "gv" "Bt" false)
+            (is_in_range_test
+               (Package.Version.of_string "3.12.0")
+               (Some (Package.Version.of_string "10.0.0"))
+               (Some (Package.Version.of_string "2.97.2"))
+               false)
```

Comment:
@dinakajoy , this is was I'm referring to in [my comment above](https://github.com/ocaml/v3.ocaml.org-server/pull/78#discussion_r704276058): if you'd like to, you could now add another test for `is_in_range_test` testing something along the lines
```
            (is_in_range_test
               (Package.Version.of_string "3.12.0")
               None
               (Some (Package.Version.of_string "3.13.0"))
               true)
```
But as I was saying, it's not necessary to add that.

Reply:
Sure, so I added it this way: 
```
    ; ( "test that is_in_range function checks that current version is not \
         within the start and last version range specified"
      , [ Alcotest.test_case
            "and returns false"
            `Quick
            (is_in_range_test
               (Package.Version.of_string "3.12.0")
               (Some (Package.Version.of_string "10.0.0"))
               (Some (Package.Version.of_string "2.97.2"))
               false)
        ] )
    ; ( "test that is_in_range function checks that current version is within \
         the start and last version range specified"
      , [ Alcotest.test_case
            "and returns false"
            `Quick
            (is_in_range_test
               (Package.Version.of_string "3.12.0")
               None
               (Some (Package.Version.of_string "3.13.0"))
               true)
        ] )```